web-client/src/containers/NewRequestContainer/NewRequestContainer.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Maybe change the name of this file to NewRequestFormContainer.
@jpmarques66 opinion?

Reply:
Yes agreed, especially because this is where we'll handle form submission.

Reply:
Done here - https://github.com/reach4help/reach4help/pull/247/commits/f19e1342f61aa5199f077b5774fddf74d8326d2c